crates/plex-api/src/media_container/server/library.rs

```diff
@@ -442,7 +460,8 @@ pub struct ServerLibrary {
     pub pivots: Vec<Pivot>,
     pub agent: String,
     pub hub_key: String,
-    pub id: String,
+    #[serde(default, deserialize_with = "deserialize_number_from_string")]
+    pub id: u32,
```

Comment:
I'm not sure about this change — the movies and tv shows on Plex (their streaming) have alpha-numeric IDs. E.g. `5df3ede26fd84c001ddc1586` is [RoboCop: Prime Directives (2001)](https://app.plex.tv/desktop/#!/provider/tv.plex.provider.vod/details?key=%2Flibrary%2Fmetadata%2F5df3ede26fd84c001ddc1586&context=library%3Ahub.movies.sci-fi_tv~16~0). Things in the watchlist also have similar IDs.

On the other hand, currently, we have nothing that would've worked with Plex Online :D

Reply:
Interesting. Yeah ok I'll change it back, doesn't make a lot of difference really.

Reply:
Actually I just realised I made that change when I was trying to do something else that ended up not working, so it is actually unnecessary anyway 🤦